### DISCUSSION TITLE
DNS turns off my location

### DISCUSSION THREAD

**Discussion by ch0coopie** [msasanmh/DNSveil#222] [Q&A] [answered]
Thank you for the great developing this project,but i just got into some problem and i dont know how to solve it cause im new to those things.

DNS turn off my location and somehow the bar goes greyed,cant figure how to turn on my location back,

![Screenshot 2024-12-29 230945](https://github.com/user-attachments/assets/4ddc7b1a-4c31-4dda-8363-c5ebe57ddad0)

ch0co
Thanks

**Comment by msasanmh** [comment, marked as answer]
> Thank you for the great developing this project,but i just got into some problem and i dont know how to solve it cause im new to those things.
> 
> DNS turn off my location and somehow the bar goes greyed,cant figure how to turn on my location back,
> 
> ![Screenshot 2024-12-29 230945](https://github.com/user-attachments/assets/4ddc7b1a-4c31-4dda-8363-c5ebe57ddad0)
> 
> ch0co
> Thanks
> 
> 

"Managed by your organization" means some of your settings have been modified through Group Policy or Registry. 

DNSveil never changes your Group Policy or Registry settings.

Some apps can do this: Antivirus, Security softwares, Tweak apps and etc.

But it's easy to fix. Read this page:
https://www.maketecheasier.com/fixes-some-settings-managed-organization-error-windows/#location-services-registry
